package.json

```diff
@@ -202,6 +202,7 @@
     "unleash-client": "^3.12.0",
     "unstated": "^2.1.1",
     "use-cursor": "^1.2.3",
+    "use-debounce": "^10.0.2",
```

Comment:
As reluctant as I am to add a dependency for something simple... [this actually isn't simple to do correctly](https://github.com/xnimorz/use-debounce/blob/master/src/useDebouncedCallback.ts) in React.